rest-server/core/src/main/scala/com/actionml/core/store/Dao.scala

```diff
@@ -38,10 +39,11 @@ trait AsyncDao[T] {
   def insertManyAsync(c: Seq[T])(implicit ec: ExecutionContext): Future[Unit]
   def updateAsync(filter: (String, QueryCondition)*)(o: T)(implicit ec: ExecutionContext): Future[T]
   def saveOneByIdAsync(id: String, o: T)(implicit ec: ExecutionContext): Future[Unit]
-  def saveOneAsync(o: T)(implicit ec: ExecutionContext): Future[Unit]
```

Comment:
This API makes no sense. It seems good to remove. You cannot save without an ID or query.

Reply:
one can insert without id or query. def saveOne is useful